.ci/setup_env_ubuntu.sh

```diff
@@ -63,6 +65,11 @@ main()
 		done
 	fi
 
+	# packages for rust agent, build on 18.04 or later
+	if [ "${VERSION_ID}" >= "18.04" ]; then
```

Comment:
I don't understand - if building on an older version of Ubuntu, you don't need these packages? Do you mean we should exit the script with an error if version `<= 18.04`?

Also, this test doesn't work. You need to do something like one of the following:

```sh
result=$(echo "$VERSION_ID > 18.04" | bc)
```

or,

```sh
int_version=$(echo $VERSION_ID|tr -d \.)

[ $int_version -ge 1804 ] && ...
```

Reply:
Some packages are missed in older version of ubuntu. They are needed for build rust agent. I think the string comparison is ok

Reply:
-_-. string comparison has no >=. The ! version < "18.04" works

Reply:
fxied